.generator/cli.py

```diff
@@ -398,22 +398,35 @@ def _create_repo_metadata_from_service_config(
         Dict: The content of the .repo-metadata.json file.
     """
     full_service_config_path = f"{source}/{api_path}/{service_config_name}"
+    with open(full_service_config_path, "r") as f:
+        service_config = yaml.safe_load(f)
+
+    publishing = service_config.get("publishing", {})
+    name_pretty = service_config.get("title", "")
+    product_documentation = publishing.get("product_documentation", "")
```

Comment:
Do you mean `documentation_uri`?

```suggestion
    product_documentation = publishing.get("documentation_uri", "")
```
https://github.com/googleapis/googleapis/blob/68bc915cb97cdd13347215377f2ca997dff245ad/google/api/client.proto#L185